### PR TITLE
upgrade to Symfony 6.4 | 7.1

### PR DESCRIPTION
**Pull request by vitek-rostislav**
As we use this bundle in Shopsys Platform and we plan to upgrade the platform to the latest Symfony, we must first add the support here.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,14 +6,26 @@
...
-    "jest": "^25.1.0"
+    "jest": "^25.1.0",
+    "@hotwired/stimulus": "^3.0.0",
+    "@symfony/stimulus-bridge": "^3.2.0",
+    "@symfony/webpack-encore": "^4.0.0",
+    "core-js": "^3.23.0",
+    "regenerator-runtime": "^0.13.9",
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0",
+    "webpack-notifier": "^1.15.0"
```
Comment (TomasLudvik) [inline package.json]: I wonder if these should not be updated as well

File: composer.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
         "symfony/twig-bundle": "*",
         "symfony/webpack-encore-bundle": "^1.4",
         "symfony/yaml": "*",
-        "tgalopin/html-sanitizer-bundle": "^1.1",
+        "symfony/html-sanitizer": "*",
```
Comment (TomasLudvik) [inline composer.json:39]: I do not feel good about having dependency version as `*`.

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (vitek-rostislav) [reply composer.json:39]: On a call, we agreed on keeping it as is

File: package.json (hunk 1 above)
Comment (vitek-rostislav) [reply package.json]: I reverted the unneeded changes in `package.json` and updated all the existing dependencies to their latest versions :wink:

**Review by TomasLudvik — APPROVED** [review]

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**